README.md

```diff
@@ -28,6 +42,63 @@
   array.map(iteratee)
   ```
 
+#### Use a predicate to accumulate a single value from an array (left-to-right)
```

Comment:
I understand "predicate" to mean a function with a boolean value. Just "function" would be better. Otherwise looks good.